drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -266,6 +273,7 @@ end
 ---@return HueApiKeyResponse[]? api_key_response nil on err
 ---@return string? error nil on success
 ---@return string? partial partial response if available, nil otherwise
+---@return ...
```

Comment:
nit: what is this indicating, that the function could return any number of additional `any` typed arguments? What is the advantage of adding this instead of enumerating the actual return types after the two know error values?

Reply:
This was to make the typechecker happy. The actual `unpack` here should stop at the three things indicated, but the signature of `table.unpack` in `LuaLS`'s meta typedefs indicates up to 12 return values.